packages/ilib-loctool-webos-common/test/utils.test.js

```diff
@@ -164,4 +164,137 @@ describe("utils", function() {
+                        {
+                            "_attributes" : {"type": "SoundBar"},
+                            "_text": "\"SoundBar\" 이용이 불가능합니다"
+                        }
+                    ],
+                    "_attributes": {
+                        "category": "device-type"
+                    }
+                }
+            }
+        });
+
+        var deviceType = "SoundBar"
+        var result = utils.getTarget(translated, deviceType);
+        expect(result).toBe("\"SoundBar\" 이용이 불가능합니다");
+    });
+    test("test_getTarget2", function() {
+        expect.assertions(1);
+
+        var translated = new ResourceString({
+            id: "app",
+            sourceLocale: "en-US",
+            targetLocale: "ko-KR",
+            key: "NOT AVAILABLE",
+            source: "NOT AVAILABLE",
+            target: "이용이 불가능합니다",
+            metadata: {
+                "mda:metaGroup": {
+                    "mda:meta": [
+                        {
+                            "_attributes" : {"type": "Monitor"},
+                            "_text": "\"Monitor\" 이용이 불가능합니다"
+                        },
+                        {
+                            "_attributes" : {"type": "Box"},
+                            "_text": "\"Box\" 이용이 불가능합니다"
+                        },
+                        {
+                            "_attributes" : {"type": "SoundBar"},
+                            "_text": "\"SoundBar\" 이용이 불가능합니다"
+                        }
+                    ],
+                    "_attributes": {
+                        "category": "device-type"
+                    }
+                }
+            }
+        });
+        var result = utils.getTarget(translated);
+        expect(result).toBe("이용이 불가능합니다");
+    });
+    test("test_getTarget3", function() {
+        expect.assertions(1);
+
+        var translated = new ResourceString({
+            id: "app",
+            sourceLocale: "en-US",
+            targetLocale: "ko-KR",
+            key: "NOT AVAILABLE",
+            source: "NOT AVAILABLE",
+            target: "이용이 불가능합니다",
+            metadata: {
+                "mda:metaGroup": {
+                    "mda:meta": [
+                        {
+                            "_attributes" : {"type": "Monitor"},
+                            "_text": "\"Monitor\" 이용이 불가능합니다"
+                        },
+                        {
+                            "_attributes" : {"type": "Box"},
+                            "_text": "\"Box\" 이용이 불가능합니다"
+                        },
+                        {
+                            "_attributes" : {"type": "SoundBar"},
+                            "_text": "\"SoundBar\" 이용이 불가능합니다"
+                        }
+                    ],
+                    "_attributes": {
+                        "category": "device-type"
+                    }
+                }
+            }
+        });
+
+        var deviceType = "SoundBarrrrr"
```

Comment:
typo?

Reply:
oh it's intentional. To test case when the `deviceType` is not matched  the medata of the ResourceString